providers/aws/provider/provider.go

```diff
@@ -262,8 +262,6 @@ func (s *Service) detect(asset *inventory.Asset, conn plugin.Connection) error {
 	if c, ok := conn.(*connection.AwsConnection); ok {
 		asset.Name = c.Conf.Host
 		asset.Platform = c.PlatformInfo()
-		// TODO: do not do this here but in discovery
-		asset.PlatformIds = []string{"//platformid.api.mondoo.app/runtime/aws/accounts/" + c.AccountId()}
```

Comment:
it is done in discovery... this line is now causing trouble, so we get rid of it